packages/gatsby-remark-copy-linked-files/src/index.js

```diff
@@ -3,9 +3,19 @@ const isRelativeUrl = require(`is-relative-url`)
 const fsExtra = require(`fs-extra`)
 const path = require(`path`)
 const _ = require(`lodash`)
-const $ = require(`cheerio`)
+const cheerio = require(`cheerio`)
+const sizeOf = require(`image-size`)
```

Comment:
This new dependency `image-size` was added yet not included in the package.json which causes this plugin to fail for me